language/random.go

```diff
@@ -0,0 +1,32 @@
+package language
+
+import (
+	"errors"
+	"regexp"
+	"sort"
+	"strconv"
+)
+
+var decimal = "\\b\\d+([\\.,]\\d+)?"
+
+func FindRangeLimits(local, entry string) ([]int, error) {
+	decimalsRegex := regexp.MustCompile(decimal)
+	limitStrArr := decimalsRegex.FindAllString(entry, 2)
+	limitArr := make([]int, 0)
+	if limitStrArr != nil {
```

Comment:
You could nest fewer loops/conditions by doing:
```go
if limitStrArr == nil {
	return make([]int, 0), errors.New("No range")
}

if len(limitStrArr) != 2 {
	return nil, errors.New("Need 2 numbers, a lower and upper limit")
}

for _, v := range limitStrArr {
	num, err := strconv.Atoi(v)
	if err != nil {
		return nil, errors.New("Non integer range")
	}
	limitArr = append(limitArr, num)
}
```

And add more spaces in the code.